test/fixtures/03-device-state/device_service_environment_variables.json

```diff
@@ -5,5 +5,12 @@
 		"service": "app1_service1",
 		"name": "name_si",
 		"value": "value_si"
+	},
+	"si2_svc": {
```

Comment:
This forces the state get tests to consider the case why I added the $filter on the device expand above